packages/plugins/posthog-ui/src/lib/directives/posthog-form-track.directive.ts

```
@@ -0,0 +1,318 @@
+ *   (ngSubmit)="onSubmit()"
+ * >
+ *   <input name="email" type="email">
+ *   <input name="password" type="password">
+ *   <button type="submit">Submit</button>
+ * </form>
+ */
+@Directive({
+	selector: 'form[phFormTrack]'
+})
+export class PostHogFormTrackDirective implements OnInit {
+	@Input() phFormName: string = 'form';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)